tests/unit_tests/data_steward/tools/create_tier_test.py

```diff
@@ -288,26 +288,30 @@ def test_create_dataset(self, mock_client, mock_define_dataset,
         client.side_effects = create_datasets
 
         datasets = {
-            consts.CLEAN: self.name,
-            consts.SANDBOX: f'{self.name}_{consts.SANDBOX}',
-            consts.STAGING: f'{self.name}_{consts.STAGING}'
+            consts.CLEAN: self.dataset_name,
+            consts.SANDBOX: f'{self.dataset_name[1:]}_{consts.SANDBOX}',
+            consts.STAGING: f'{self.dataset_name}_{consts.STAGING}'
         }
 
         # Tests if incorrect parameters are given
-        self.assertRaises(RuntimeError, create_datasets, None, self.name,
-                          self.input_dataset, self.tier, self.release_tag)
+        self.assertRaises(RuntimeError, create_datasets, None,
+                          self.dataset_name, self.input_dataset, self.tier,
+                          self.release_tag)
         self.assertRaises(RuntimeError, create_datasets, client, None,
                           self.input_dataset, self.tier, self.release_tag)
-        self.assertRaises(RuntimeError, create_datasets, client, self.name,
-                          None, self.tier, self.release_tag)
-        self.assertRaises(RuntimeError, create_datasets, client, self.name,
-                          self.input_dataset, None, self.release_tag)
-        self.assertRaises(RuntimeError, create_datasets, client, self.name,
-                          self.input_dataset, self.tier, None)
+        self.assertRaises(RuntimeError, create_datasets, client,
+                          self.dataset_name, None, self.tier, self.release_tag)
+        self.assertRaises(RuntimeError, create_datasets, client,
+                          self.dataset_name, self.input_dataset, None,
+                          self.release_tag)
+        self.assertRaises(RuntimeError, create_datasets, client,
+                          self.dataset_name, self.input_dataset, self.tier,
+                          None)
 
         # Test
-        expected = create_datasets(client, self.name, self.input_dataset,
-                                   self.tier, self.release_tag)
+        expected = create_datasets(client, self.dataset_name,
```

Comment:
NIT: this variable should probably be called actual

Reply:
You're right! I changed it